content/usage/advanced/index.md

```diff
@@ -1,162 +1,342 @@
-{{ easy_image(src="view-config", width=250, center=true) }}
-- The "Current widgets" list allows
-    1. dragging the widgets in the current view to reorder which widget is on top
+{{ easy_image(src="view-config", width=400, center=true) }}
+- The "Widgets in View" list allows
+    - hovering over a widget name to highlight its display in the interface
+    - dragging the widgets in the main view area to reorder which widget is on top
         - This helps for use-cases like overlaying a HUD element on a video display
-    1. removing an existing widget from the current view
-    1. resizing a widget to fill the entire view
+    - removing an existing widget from the current view
+    - resizing a widget to fill the entire view
+{{ easy_image(src="view-widgets", width=300, center=true) }}
 - New widgets can be added via the bottom section
-    - Clicking on a regular widget adds it to the view, after which it can be positioned and resized as desired
+    - Dragging a regular widget card into the main display area adds it to the View, after which it can be 
+      positioned and resized as desired
     - Mini widgets have fixed sizes, but can be dragged and dropped into the desired location in the header/footer
       bars or in a custom [mini widget bar](#mini-widget-bar)
-    - The selector in the bottom left can be used to choose between editing regular or mini widgets
-- Some widgets can be configured, by clicking the cog settings icon in the "Current widgets" list
+        - The header bar is shared between Views, and the bottom bar is unique to each View
+    - The selector in the bottom left can be used to choose between editing regular, mini, or input widgets
+- Some widgets can be configured, by clicking the cog settings icon in the "Widgets in View" list
     - [There are currently cog icons for all widgets](https://github.com/bluerobotics/cockpit/issues/541),
       so if you click a cog icon and nothing happens it means that widget is not configurable
 
 ### Widgets
 
-There are several types of widgets available, and in future it will be possible to create, import, and use custom widgets as well.
+There are several types of widgets available, including different displays of the same information for use in different contexts:
+- **Regular Widgets** include detailed displays like videos, maps, instrument gauges, and overlays
+   - They can only be located in the main display area
+   - Their positions and sizes snap to the [Profile alignment grid](#profile-configuration), if it is enabled
+- **Mini Widgets** are small (usually text-based) indicators and buttons
+   - They can be in the shared header bar, or in the View-specific footer bar or a [mini widget bar](#mini-widget-bar)
+     in the main display area
+- [**Input Widgets**](#input-widgets) are like mini-widgets dedicated for custom user inputs / commands
+   - They need to be placed inside a [custom widget base](#custom-widget-base) widget
```

Comment:
They can actually be placed in any mini-widget container, like the header/footer bars and the MiniWidgetBars.